src/python/bot/fuzzers/engine_common.py

```diff
@@ -281,19 +282,28 @@ def dump_big_query_data(stats, testcase_file_path, fuzzer_command):
   fuzzer_stats.TestcaseRun.write_to_disk(testcase_run, testcase_file_path)
 
 
-def find_fuzzer_path(build_directory, fuzzer_name):
+def find_fuzzer_path(build_directory, engine_name, target_name):
```

Comment:
Rather than passing `engine_name`, can we just call it `blackbox` since the value doesn't matter beyond it being 'blackbox'? This function shouldn't need to know about the names of the implemented engines either (as that's an implementation detail). We can also give it a default value of False and make it easier for non-blackbox callers to use.

Reply:
That makes sense. I was a bit put off by this initially since it seemed error prone for callers to have to remember to do this, but looking at the usage it seems unlikely that we'll really be adding new calls where this matters.